web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,167 @@
 ---
+id: "creating-new-drupal-project"
+title: "Creating a New Drupal Project"
+slug: "/Backend Starters/Decoupled Drupal/Creating a New Project"
 sidebar_position: 2
 ---
 
-# Creating a New Project
+## Choosing an Approach
 
-## Prerequisites
+### Use Build Tools if:
+
+- Testing is an important part of your workflow
+
+- You don’t want to manually push changes to your code repo.
+
+### Use Dashboard Upstream if:
+
+- You require a simpler setup
+
+- Pantheon repo is your source of truth
+
+### Installing using Dashboard Upstream
+
+- Create an empty upstream site one of two ways:
+
+  - Via the Pantheon Dashboard at this link:
+
+    - [Empty Upstream](https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65)
+
+  - Or Alternatively via Terminus:
+
+    ```
+    terminus site:create my-new-site "Describe Site" empty --org='My Team Name'
+    ```
+
+  - Replace `'{My Team Name}'` with your team name - for example `My Agency`. This can also be omitted.
+
+#### In the Pantheon Admin Dashboard:
+
+Switch **Development Mode** to **Git**
+
+Click the **Upgrade** button, and change the site plan to **Performance** > **Medium**.
+
+\*\* Clone your site's repository on your local
+
+\*\* Change into the newly cloned directory
+
+\*\* Add remote for the decoupled upstream: `git remote add upstream https://github.com/pantheon-systems/decoupled-drupal-composer-managed`
+
+\*\* Checkout the main branch from upstream:
+
+```
+
+git fetch upstream main
+
+git checkout -t upstream/main
+
+```
+
+\*\* Reset the master branch to match main, then force push that to the Dev environment:
+
+```
+
+git branch -D master
+
+git checkout -b master
+
+git push origin master --force
+
+```
+
+:::Note
```

Comment:
This doesn't look to be rendering as expected for some reason. Capital letter perhaps? `:::info` seems to work elsewhere in the codebase.